src/frontend/utils/tests/imageSnapshot.tsx

```diff
@@ -0,0 +1,44 @@
+import { cleanup, render } from '@testing-library/react';
+import { generateImage } from 'jsdom-screenshot';
+import { Grommet } from 'grommet';
+import React from 'react';
+import path from 'path';
+
+import { wrapInIntlProvider } from './intl';
+import { wrapInRouter } from './router';
+import { theme } from '../theme/theme';
+import { GlobalStyles } from '../theme/baseStyles';
+
+export const imageSnapshot = async (width?: number, height?: number) => {
+  width = width || 800;
+  height = height || 600;
+  const screenshot = await generateImage({ viewport: { width, height } });
+  expect(screenshot).toMatchImageSnapshot({
+    customDiffDir: path.resolve(__dirname, '../../__diff_output__'),
+    failureThreshold: 0.01,
+    failureThresholdType: 'percent',
+  });
+};
+
+export const renderImageSnapshot = async (
+  component: JSX.Element,
+  width?: number,
+  height?: number,
+) => {
+  cleanup();
+  render(
+    wrapInIntlProvider(
+      wrapInRouter(
```

Comment:
Both return a `JSX.Element`. Why do you need to call them here ? I think it's the developer responsability to use them directly in the test.

Moreover, maybe I need to configure both `wrapper` functions and the way you made it is not possible.

Reply:
Indeed they are hardcoded right now.
Can you give me an usage example you would like to have, @lunika ?